WorldWideWeb/NXShims.h

```diff
@@ -0,0 +1,85 @@
+//
+//  NXShims.h
+//  WorldWideWeb
+//
+//  Created by Chris Brandow on 8/23/24.
+//
+
+#ifndef NXShims_h
+#define NXShims_h
+
+
+#endif /* NXShims_h */
```

Comment:
Nit: This header guard should probably be moved to the end of the file.

Probably not really needed anyway, since `#import` will take care of only including headers once, but I've heard that it's good style to either use header guards or `#pragma once`. At some point we should probably pick one and apply it consistently throughout the project.